src/SussolReactTable.jsx

```diff
@@ -42,24 +81,27 @@ export class SussolReactTable extends React.Component {
     });
   }
 
-// Renders column headers. Gets the sort direction from state and the label columns array.
+  // Renders column headers. Gets the sort direction from state and the label columns array.
   renderColumnHeader(column) {
     let sortIcon;
 
-    if ( column.sortable &&this.state.sortBy === column.key ) {
-      sortIcon = (
-        <FontIcon className="material-icons">
-          {this.state.isAscending ? 'arrow_drop_up' : 'arrow_drop_down'}
-        </FontIcon>
-      );
+    if (column.sortable && this.state.sortBy === column.key) {
+      sortIcon = this.state.isAscending
+        ? (this.renderSortIcon('asc'))
+        : (this.renderSortIcon('desc'));
     }
+
     return (
-      <ColumnHeaderCell >
+      <ColumnHeaderCell>
         <FlatButton
           label={column.title}
           labelPosition="before"
           icon={sortIcon}
-          onClick={() => this.toggleSortOrder(column.key)}
+          onClick={
+            column.sortable
+            ? () => this.toggleSortOrder(column.key)
+            : noop
```

Comment:
null?

Reply:
Probs wrote this too fast. `null` should work, I believe.

Reply:
Just checked. It's b/c Material UI `EnhancedButton` wants a func prop sent down, or it throws.